TOREVIVE.md

```diff
@@ -0,0 +1,37 @@
+## Goal
+
+* Ruby 2.0 and above only
+* ruote_sequel with Aurora & MySQL only
```

Comment:
I don't have a context for this document, but this already sounds like we're putting ourselves in a place we shouldn't be, ex Tim's effort on converting TP to postgres had highly significant performance improvements.